tests/clients/test_dapr_grpc_client.py

```diff
@@ -681,6 +682,58 @@ def test_unlock_input_validation(self):
             with self.assertRaises(ValueError):
                 dapr.unlock(store_name, resource_id, invalid_input)
 
+    #
+    # Tests for workflow
+    #
+
+    def test_workflow(self):
+        dapr = DaprGrpcClient(f'localhost:{self.server_port}')
+        # Sane parameters
+        workflow_name = 'testWorkflow'
+        event_name = 'eventName'
+        instance_id = str(uuid.uuid4())
+        workflow_component = "dapr"
+        input = "paperclips"
+        event_data = "cars"
+        # Start the workflow
+        start_response = dapr.start_workflow(instance_id, workflow_name, workflow_component,
+                                             input.encode('utf-8'), None)
+        self.assertEqual(instance_id, start_response.instance_id)
+
+        # Get info on the workflow to check that it is running
+        get_response = dapr.get_workflow(instance_id, workflow_component)
+        self.assertEqual(WorkflowRuntimeStatus.RUNNING, get_response.runtime_status)
+
+        # Pause the workflow
+        dapr.pause_workflow(instance_id, workflow_component)
+
+        # Get info on the workflow to check that it is paused
+        get_response = dapr.get_workflow(instance_id, workflow_component)
+        self.assertEqual(WorkflowRuntimeStatus.SUSPENDED, get_response.runtime_status)
+
+        # Resume the workflow
+        dapr.resume_workflow(instance_id, workflow_component)
+
+        # Get info on the workflow to check that it is resumed
+        get_response = dapr.get_workflow(instance_id, workflow_component)
+        self.assertEqual(WorkflowRuntimeStatus.RUNNING, get_response.runtime_status)
+
+        # Raise an event on the workflow. TODO: Figure out how to check/verify this
```

Comment:
Regarding this TODO, consider putting some data in the workflow response properties collection to let you know that the raise_event call succeeded.